Cargo.toml

```diff
@@ -12,8 +12,12 @@ keywords = ["cargo", "temporary", "project", "cli", "dependencies"]
 categories = ["development-tools::cargo-plugins"]
 include = ["src/**/*.rs", "README.md", "LICENSE.Apache-2.0", "LICENSE.MIT"]
 
+[features]
+generate = ["cargo-generate"]
+
 [dependencies]
 anyhow = "1.0"
+cargo-generate = { version = "0.14", path = "../cargo-generate", optional = true }
```

Comment:
I use `path = "../cargo-generate"` because I use a version based on this [PR](https://github.com/cargo-generate/cargo-generate/pull/624)